includes/class-sensei-quiz.php

```diff
@@ -1604,7 +1604,7 @@ public static function output_quiz_hidden_fields() {
 
 		foreach ( $sensei_question_loop['questions_asked'] as $question_id ) {
 			?>
-			<input type="hidden" name="questions_asked[]" value="<?php echo esc_attr( $question_id ); ?>">
+			<input type="hidden" name="questions_asked[]" form="sensei-quiz-form" value="<?php echo esc_attr( $question_id ); ?>">
```

Comment:
Oh that method and its siblings are actually no longer used, just forgot to remove them. Most of the HTML here are actually not course theme specific so it wouldn't be a good place anyway.  And it's simpler not to introduce a getter/constant for this, since we don't have ones for all the other similar things like the action names.